src/components/layout/nav/Navbar.jsx

```diff
@@ -145,14 +188,23 @@ const Navbar = () => {
   const handleClickAway = () => {
     handleStatus("CLICK_AWAY");
   };
+  let location = useLocation();
+
+  useEffect(() => {
+    changeNavbarClassNameForSpecificComponents(location.pathname);
+    changeNavbarClassNameOnScroll(location.pathname);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [location.pathname, scrollState]);
 
   return (
     <>
       <ClickAwayListener onClickAway={handleClickAway}>
         {isSearchBoxOpen && <SearchBox />}
         <Container
           fluid
-          className={`navbar ${isSearchBoxOpen && "moveOverHeader"}`}
+          className={`${navbarClassName} navbar ${
+            isSearchBoxOpen && "moveOverHeader"
```

Comment:
i'm not sure this && thing works for classnames. when it's false, does it add the classname false?

Reply:
Yes, it's working and adding the classname `false` : 
![image](https://user-images.githubusercontent.com/30778803/90004376-31670500-dc9e-11ea-8759-35bc932bdbc6.png)

Reply:
i don't think that counts as "working" :) that would be a bug to add a class name that does nothing.

please change this so that it doesn't use && here (we don't usually use it inside of the backticks like this, precisely because it creates the random "false" string)